core/src/main/java/com/sequenceiq/cloudbreak/service/externaldatabase/AzureDatabaseServerParameterSetter.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.cloudbreak.service.externaldatabase;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.cloudbreak.service.externaldatabase.model.DatabaseServerParameter;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.DatabaseServerV4StackRequest;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.azure.AzureDatabaseServerV4Parameters;
+
+@Component
+public class AzureDatabaseServerParameterSetter implements DatabaseServerParameterSetter {
+
+    @Value("${cb.azure.externaldatabase.retentionperiod:7}")
+    private int retentionPeriod;
+
+    @Value("${cb.azure.externaldatabase.geoRedundantBackup:true}")
+    private Boolean geoRedundantBackup;
+
+    @Override
+    public void setParameters(DatabaseServerV4StackRequest request, DatabaseServerParameter serverParameter) {
```

Comment:
`DatabaseServerParameter serverParameter` parameter is not used. Are you sure this is the right method signature?

Reply:
Azure has no specifics yet. AWS has. Still, this also has to go into a unified child class.